recipes/gbquality/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "gbquality" %}
+{% set version = "0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: dac25703bff5c993091c30c4c53f4d2c61e83d1a9b24fe2f2c3d267857b3a966
+
+build:
```

Comment:
I believe this can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python